File: ExerciciosJavaScript(LógicadeProgramacao)/exercicios pag 66(Manzano)/js/exercicioK.js
/*k) Elaborar um programa que apresente como resultado o valor do fatorial dos valores ímpares
// situados na faixa numérica de 1 a 10 */


var valor = 0
var antecessor = 0
var resultado = 0
var fatorial = 0
var guardaValor = 0
var contadora = 1


for (contadora = 1; contadora <= 10; contadora++) {
    antecessor = contadora
    fatorial = contadora

    if ((contadora % 2) == 1) {
        alert("O valor "+ contadora+ " é impar")

        do {
            antecessor = antecessor - 1
            fatorial = fatorial * antecessor
            resultado = fatorial
            alert(resultado)
        } while (antecessor > 2)

        guardaValor = guardaValor + resultado

    }
}

alert("A soma do fatorial de todos os numeros digitado é " + guardaValor)

File: LogicaDeProgramacaoEclipse(Java)/LogicaProgramacaoJava(Eclipse)/exerciciosJava/src/manzanoPag66/ExercicioK.java
package manzanoPag66;

/*k) Elaborar um programa que apresente como resultado o valor do fatorial dos valores ímpares
//situados na faixa numérica de 1 a 10 */

public class ExercicioK {

	public static void main(String[] args) {


		int antecessor = 0;
		int resultado = 0;
		int fatorial = 0;
		int guardaValor = 0;
		int contadora = 1;

		for (contadora = 1; contadora <= 10; contadora++) {
			antecessor = contadora;
			fatorial = contadora;

			if ((contadora % 2) == 1) {
				System.out.println("O valor " + contadora + " é impar");

				do {
					antecessor = antecessor - 1;
					fatorial = fatorial * antecessor;
					resultado = fatorial;
					System.out.println(resultado);
				} while (antecessor > 2);

				guardaValor = guardaValor + resultado;

			}
		}

		System.out.println("A soma do fatorial de todos os numeros digitado é " + guardaValor);

	}

}
